.cirrus.yml

```diff
@@ -4,12 +4,17 @@ auto_cancellation: false
 env:
   PATH: ${HOME}/.local/bin:${PATH}
   # ^  add user paths
-  COVERALLS_REPO_TOKEN: ENCRYPTED[7ada9fe1105610d2b5ac314f5eb4cc2fea3d42a5fa1e1bfa263000277146022b71b80f7d2f00dc6058e08bd1c9034be7]
   PIP_CACHE: ${HOME}/.cache/pip
   PRE_COMMIT_HOME: ${HOME}/.cache/pre-commit
   LC_ALL: C.UTF-8
   LANG: C.UTF-8
   COVERAGE: NO
+  CI_NAME: cirrus-ci
+  CI_BRANCH: ${CIRRUS_BRANCH}
+  CI_PULL_REQUEST: ${CIRRUS_PR}
+  CI_BUILD_NUMBER: ${CIRRUS_BUILD_ID}
+  CI_BUILD_URL: https://cirrus-ci.com/build/${CIRRUS_BUILD_ID}
```

Comment:
[New feature I contributed to coveralls-python](https://github.com/TheKevJames/coveralls-python/pull/300) improving the quality of the reports.

(I also think this allows coveralls github bot to post a comment in the PR with coverage changes).